locales/en-US/server.ftl

```diff
@@ -0,0 +1,214 @@
+shotPageSelectTime = Select time
+shotPageKeepIndefinitely = Indefinitely
+shotPageKeepTenMinutes = 10 Minutes
+shotPageKeepOneHour = 1 Hour
+shotPageKeepOneDay = 1 Day
+shotPageKeepOneWeek = 1 Week
+shotPageKeepTwoWeeks = 2 Weeks
+shotPageKeepOneMonth = 1 Month
+shotPageSaveExpiration = save
+shotPageCancelExpiration = cancel
+shotPageDoesNotExpire = does not expire
+// Note: {$timediff} is a placeholder for a future relative time clause, like "in 1 week" or "tomorrow"
+shotPageExpiresIn = expires {$timediff}
+// Note: {$timediff} is a placeholder for a past relative time clause, like "1 week ago" or "yesterday"
+shotPageExpired = expired {$timediff}
+timeDiffJustNow = just now
+timeDiffMinutesAgo = { $number ->
+  [one] 1 minute ago
+  *[other] {$number} minutes ago
+}
+timeDiffHoursAgo = { $number ->
+  [one] 1 hour ago
+  *[other] {$number} hours ago
+}
+timeDiffDaysAgo = { $number ->
+  [one] yesterday
+  *[other] {$number} days ago
+}
+timeDiffFutureSeconds = in a few seconds
+timeDiffFutureMinutes = { $number ->
+  [one] in 1 minute
+  *[other] in {$number} minutes
+}
+timeDiffFutureHours = { $number ->
+  [one] in 1 hour
+  *[other] in {$number} hours
+}
+timeDiffFutureDays = { $number ->
+  [one] tomorrow
+  *[other] in {$number} days
+}
+
+// Shotindex page
+
+// {$status} is a placeholder for an HTTP status code, like '500'.
+// {$statusText} is a text description of the status code, like 'Internal server error'.
+shotIndexPageErrorDeletingShot = Error deleting shot: {$status} {$statusText}
+// {$searchTerm} is a placeholder for text the user typed into the search box
+shotIndexPageSearchResultsTitle = My Shots: search for {$searchTerm}
+// {$error} is a placeholder for a non-translated error message that could be shared
+// with developers when debugging an error.
+shotIndexPageErrorRendering = Error rendering page: {$error}
+shotIndexPageSearchPlaceholder
+  .placeholder = Search my shots
+shotIndexPageSearchButton
+  .title = Search
+shotIndexPageNoShotsMessage = No saved shots.
+shotIndexPageNoShotsInvitation = Go on, create some.
+shotIndexPageLookingForShots = Looking for your shots…
+shotIndexPageNoSearchResultsIntro = Hmm
+shotIndexPageNoSearchResults = We canʼt find any shots that match your search.
+shotIndexPageClearSearchButton
+  .title = Clear search
+shotIndexPageConfirmShotDelete = Delete this shot?
+
+// Metrics page
+// Note: all metrics strings are optional for translation
+
+// Note: 'Firefox Screenshots' should not be translated
+metricsPageTitle = Firefox Screenshots Metrics
+metricsPageTotalsQueryTitle = Totals
+metricsPageTotalsQueryDescription = An overview of Screenshots
+metricsPageTotalsQueryDevices = Total devices registered
+metricsPageTotalsQueryActiveShots = Active shots
+metricsPageTotalsQueryExpiredShots = Expired (but recoverable)
+metricsPageTotalsQueryExpiredDeletedShots = Expired (and deleted)
+metricsPageShotsQueryTitle = Shots By Day
```

Comment:
Not a native speaker, but is this proper title case? I assume "by" should be lowercase.